components/Answer.js

```diff
@@ -3,7 +3,9 @@ import { useState, useEffect, useRef } from "react";
 
 const InputFieldLayout = styled.div`
   display: flexbox;
```

Comment:
better change this to `display: flex` before CJ sees it

Reply:
Ohhh... right!